docs/openapi/components/schemas/common/PGAStatusMessageCertificate.yml

```diff
@@ -0,0 +1,110 @@
+$linkedData:
+  term: PGAStatusMessageCertificate
+  '@id': https://w3id.org/traceability#PGAStatusMessageCertificate
+title: PGA Status Message Certificate
+description: Certificate Issued by a PGA for the status of Cargo
+type: object
+properties:
+  '@context':
+    type: array
+    const:
+      - 'https://www.w3.org/2018/credentials/v1'
+      - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    const:
+      - VerifiableCredential
+      - PGAStatusMessageCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    title: Credential Subject
+    description: List of PGA Status Messages pertaining to per-line items
+    type: array
+    items:
+      $ref: ./PGAStatusMessage.yml
+  proof:
+    type: object
+  relatedLink:
+    title: Related Link
+    description: Links related to this verifiable credential
+    type: array
+    items:
+      $ref: ./LinkRole.yml
+additionalProperties: false
+required:
+  - type
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "https://example.com/credential/123",
+    "type": [
+      "VerifiableCredential",
+      "PGAStatusMessageCertificate"
+    ],
+    "name": "PGA Hold Certificate",
+    "description": "Certificate Issued by a PGA for the status of Cargo",
+    "relatedLink": [],
+    "issuanceDate": "2019-12-11T03:50:55Z",
+    "issuer": {
+      "id": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+      "type": "Organization",
+      "name": "Waters Inc",
+      "description": "",
+      "address": {
+        "type": "PostalAddress",
+        "streetAddress": "027 Brakus Knoll",
+        "addressLocality": "East Johnniemouth",
+        "addressRegion": "Arizona",
+        "postalCode": "25780-5840",
+        "addressCountry": "Grenada"
+      },
+      "email": "Kendrick.Spinka57@example.org",
+      "phoneNumber": "555-322-9464",
+      "faxNumber": "555-766-1744"
+    },
+    "credentialSubject": [
+      {
+        "type": "PGAStatusMessage",
+        "recordNo": "SO70",
+        "entryNo": "AAA-1234567-8",
+        "entryLineSequence": "L1:S1",
+        "statusCode": "O2",
+        "statusCodeDescription": "Hold Intact",
+        "validCodeReason": "25",
+        "validCodeReasonDescription": "Additional Verification Needed",
+        "subReasonCode": "132",
+        "subReasonCodeDescription": "Documentation Needed"
+      },
+      {
```

Comment:
Is this to be understood as the second message which is sent about the same shipment? And the first message is just kept around, but would have been sent already? 
Typically this would include a timestamp of some sort. (But I don't want to invent non-existing requirements).

Reply:
My guess for this scenario is that PGA's are likely going to be managing thousands of shipments. And each record is likely going to contain multiple line numbers. So it would make sense to allow for multiple messages for respective records and line numbers to be exchanged in a single credential.